Build failures with mtl-2.3

```
Building library for cue-sheet-2.0.1..
[1 of 4] Compiling Text.CueSheet.Types ( Text/CueSheet/Types.hs, dist/build/Text/CueSheet/Types.o, dist/build/Text/CueSheet/Types.dyn_o )

Text/CueSheet/Types.hs:41:1: warning: [-Wunused-imports]
    The import of ‘Data.Monoid’ is redundant
      except perhaps to import instances from ‘Data.Monoid’
    To import instances alone, use: import Data.Monoid()
   |
41 | import Data.Monoid ((<>))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^
[2 of 4] Compiling Text.CueSheet.Render ( Text/CueSheet/Render.hs, dist/build/Text/CueSheet/Render.o, dist/build/Text/CueSheet/Render.dyn_o )
[3 of 4] Compiling Text.CueSheet.Parser ( Text/CueSheet/Parser.hs, dist/build/Text/CueSheet/Parser.o, dist/build/Text/CueSheet/Parser.dyn_o )

Text/CueSheet/Parser.hs:159:3: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) a10
    |
159 |   void (many pHeaderItem)
    |   ^^^^

Text/CueSheet/Parser.hs:160:3: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) a9
    |
160 |   void (some pFile)
    |   ^^^^

Text/CueSheet/Parser.hs:229:3: error:
    Variable not in scope:
      void
        :: Parser ByteString -> StateT Context (Parsec Eec ByteString) a3
    |
229 |   void (symbol "REM")
    |   ^^^^

Text/CueSheet/Parser.hs:234:3: error:
    Variable not in scope:
      void
        :: Parser ByteString -> StateT Context (Parsec Eec ByteString) a8
    |
234 |   void (symbol "FILE")
    |   ^^^^

Text/CueSheet/Parser.hs:243:3: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) a7
    |
243 |   void (some (pTrack <|> pRem))
    |   ^^^^

Text/CueSheet/Parser.hs:255:3: error:
    Variable not in scope:
      void
        :: Parser ByteString -> StateT Context (Parsec Eec ByteString) a6
    |
255 |   void (symbol "TRACK")
    |   ^^^^

Text/CueSheet/Parser.hs:283:5: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) a5
    |
283 |     void (many pTrackHeaderItem)
    |     ^^^^

Text/CueSheet/Parser.hs:297:5: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) a4
    |
297 |     void (some (grabIndex <|> pRem))
    |     ^^^^

Text/CueSheet/Parser.hs:301:5: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) (Maybe ())
           -> StateT Context (Parsec Eec ByteString) ()
    |
301 |     void (optional pPostgap)
    |     ^^^^

Text/CueSheet/Parser.hs:317:3: error:
    Variable not in scope:
      void
        :: StateT Context (Parsec Eec ByteString) [()]
           -> StateT Context (Parsec Eec ByteString) ()
    |
317 |   void (some pFlag) <* eol <* scn
    |   ^^^^

Text/CueSheet/Parser.hs:406:3: error:
    Variable not in scope:
      void
        :: Parser ByteString -> StateT Context (Parsec Eec ByteString) a2
    |
406 |   void (symbol "INDEX")
    |   ^^^^

Text/CueSheet/Parser.hs:417:3: error:
    Variable not in scope:
      void :: m2 (Token s2) -> StateT Context (Parsec Eec ByteString) a1
    |
417 |   void (char 58)
    |   ^^^^

Text/CueSheet/Parser.hs:427:3: error:
    Variable not in scope:
      void :: m1 (Token s1) -> StateT Context (Parsec Eec ByteString) a0
    |
427 |   void (char 58)
    |   ^^^^

Text/CueSheet/Parser.hs:458:11: error:
    Variable not in scope:
      void
        :: Parser ByteString -> StateT Context (Parsec Eec ByteString) ()
    |
458 |   let p = void (symbol command)
    |           ^^^^

Text/CueSheet/Parser.hs:522:15: error:
    Variable not in scope:
      void :: m0 (Tokens s0) -> StateT Context (Parsec Eec ByteString) ()
    |
522 | sc = L.space (void $ takeWhile1P Nothing f) empty empty
    |               ^^^^
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,6 @@
-## Unreleased
+## Cue sheet 2.0.2
 
+* Builds with `mtl-2.3`.
 * Dropped support for GHC 8.2 and older.
 
 ## Cue sheet 2.0.1

--- a/Text/CueSheet/Parser.hs
+++ b/Text/CueSheet/Parser.hs
@@ -23,7 +23,8 @@ module Text.CueSheet.Parser
   )
 where
 
-import Control.Monad.State.Strict
+import Control.Monad (void)
+import Control.Monad.State.Strict (StateT, execStateT, gets, modify)
 import Data.ByteString (ByteString)
 import Data.Data (Data)
 import Data.List.NonEmpty (NonEmpty (..))

